src/librbd/operation/ResizeRequest.h

```diff
@@ -67,19 +67,15 @@ class ResizeRequest : public Request<ImageCtxT> {
    *    v
    * STATE_APPEND_OP_EVENT (skip if journaling
    *    |                   disabled)
-   *    | (unblock writes)
-   *    |
    *    |
    *    | (grow)
    *    |\--------> STATE_GROW_OBJECT_MAP (skip if object map
    *    |                 |                disabled)
    *    |                 v
-   *    |           STATE_POST_BLOCK_WRITES
-   *    |                 |
-   *    |                 v
    *    |           STATE_UPDATE_HEADER ----------------------------\
+   *    |                                 (unblock writes)          |
    *    |                                                           |
-   *    | (shrink)                                                  |
+   *    | (shrink)          (unblock writes)                        |
```

Comment:
actually, the `(no change)` branch also needs `(unblock writes)`. I thought you would put `(unblock writes)` somewhere above `(shrink)` so it could be applicable for both branches.

Reply:
Done